Orange/widgets/data/owfeatureconstructor.py

```diff
@@ -958,28 +958,5 @@ def unique(seq):
     return unique_el
 
 
-def main(argv=None):
-    from AnyQt.QtWidgets import QApplication
-    if argv is None:
-        argv = sys.argv
-    app = QApplication(list(argv))
-    argv = app.arguments()
-    if len(argv) > 1:
-        filename = argv[1]
-    else:
-        filename = "iris"
-
-    w = OWFeatureConstructor()
-    w.show()
-    w.raise_()
-    data = Orange.data.Table(filename)
-    w.setData(data)
-    w.handleNewSignals()
-    app.exec_()
-    w.setData(None)
-    w.handleNewSignals()
```

Comment:
The purpose of these `main` functions is to show the widget for visual inspection when they're being implemented or changed. Testing whether the widget survives removing the data belongs to unit tests, not here, hence I removed such code when refactoring.

If somebody feels we should try sending `None` after the widget is (manually) tested, this can be easily reimplemented in `test_run` - but I'd strongly rather not.